drivers/issi/is31fl3737.c

```diff
@@ -153,7 +154,9 @@ void IS31FL3737_init(uint8_t addr) {
 
 void IS31FL3737_set_color(int index, uint8_t red, uint8_t green, uint8_t blue) {
     if (index >= 0 && index < DRIVER_LED_TOTAL) {
-        is31_led led = g_is31_leds[index];
+        // copy the led config from progmem to SRAM
+        is31_led led;
+        memcpy_P(&led, (&g_is31_leds[index]), sizeof(led));
```

Comment:
Does not like this. 

```
drivers/issi/is31fl3737.c: In function 'IS31FL3737_set_color':
tmk_core/common/progmem.h:9:36: error: implicit declaration of function 'memcpy' [-Werror=implicit-function-declaration]
 #    define memcpy_P(dest, src, n) memcpy(dest, src, n)
                                    ^~~~~~
drivers/issi/is31fl3737.c:159:9: note: in expansion of macro 'memcpy_P'
         memcpy_P(&led, (&g_is31_leds[index]), sizeof(led));
         ^~~~~~~~
```

Reply:
Whoops, looks like `progmem.h` needs `#include <string.h>` in the else.

Reply:
I have tested with AVR and that works fine.  I have tried to setup a test compile for STM32 - but does not appear straight forward as it fails on i2c items.  

I will update progmem.h as follows unless advised otherwise.
`#pragma once

#if defined(__AVR__)
#    include <avr/pgmspace.h>
#else
#    include <string.h>
#    define PROGMEM
#    define PSTR(x) x
...`

Reply:
Can confirm that it fixes it.